src/ops/ops.go

```diff
@@ -207,3 +211,48 @@ func (o *ops) renderControllerCm() error {
 	}
 	return nil
 }
+
+func (o *ops) GetVGByPV(pvName string) (string, error) {
+	output, err := o.ExecPrivilegeCommand(o.logWriter, "vgs", "--noheadings", "-o", "vg_name,pv_name")
+	if err != nil {
+		o.log.Errorf("Failed to list VGs in the system")
+		return "", err
+	}
+
+	lines := strings.Split(output, "\n")
+	for _, line := range lines {
+		res := strings.Fields(line)
+		if len(res) < 2 {
+			continue
+		}
+
+		if strings.Contains(res[1], pvName) {
+			return res[0], nil
+		}
+	}
+	return "", nil
+}
+
+func (o *ops) RemoveVG(vgName string) error {
+	output, err := o.ExecPrivilegeCommand(o.logWriter, "vgremove", vgName, "-y")
+	if err != nil {
+		o.log.Errorf("Failed to remove VG %s, output %s, error %s", vgName, output, err)
+	}
+	return err
+}
+
+func (o *ops) RemoveLV(lvName, vgName string) error {
```

Comment:
not used?

Reply:
No, not used. Just added this for a complete set of LVM API, in case we will need it later

Reply:
hmm , not sure we need it

Reply:
less functions in interface easier life